locales/en-US/experiments.ftl

```diff
@@ -71,6 +71,8 @@ nomore404sContributors2Title = Director, Wayback Machine, The Internet Archive
 nomore404sContributors3Title = Senior Crawl Engineer, The Internet Archive
 notesDescription = A simple notepad built into Firefox. Ever need to jot something down while you browse the web? This is the experiment for you.
 notesIntroduction = Notes adds a simple, one-page notepad to your browser’s sidebar. We’ve added a few basic formatting options to start, but we want your feedback to tell us what features you’d like to see us add.
+notesLegalnotice = By proceeding, you agree to the {$terms} and {$privacy} policy of Test Pilot and {$privacylink}.
```

Comment:
I'm confused: what replaces `$terms` and `$privacy` in this string?

Also, is there a chance to have a single string for this sentence, instead of 2? If not, please add a comment explaining what replaces what.

Reply:
> $terms and $privacy are replaced by landingLegalNoticeTermsOfUse and landingLegalNoticePrivacyNotice, which are single-word strings we are borrowing from elsewhere on the site

Can you point me to the strings? Because I've searched, and failed to find them anywhere in l10n files.

Reply:
@flodolo I was able to replace that ugly implementation with a much simpler LocalizedHtml implementation. Please take a look at the new code instead.

Thank you for complaining about that!